code/game/objects/items/weapons/ranged/flintlock.dm

```diff
@@ -134,8 +134,7 @@
 		if(user.STAPER > 8)
 			BB.accuracy += (user.STAPER - 8) * 4 //each point of perception above 8 increases standard accuracy by 4.
 			BB.bonus_accuracy += (user.STAPER - 8) //Also, increases bonus accuracy by 1, which cannot fall off due to distance.
-		if(user.STAPER > 10)
-			BB.damage = BB.damage * (user.STAPER / 10)
+		BB.damage = BB.damage *1.625 // 80 * 1.625 = 130 of damage.
```

Comment:
if your intention is for the gun to have a static value unaffected by stats or other factors you're better off using a define here

Reply:
ye but the thing is not every gunpowder weaponry would have the same static damage but yes, creating a define like FLINTLOCK_MULT, MUSKET_MULT and etc could be a thing